sdks/python/apache_beam/runners/worker/sideinputs.py

```diff
@@ -78,29 +82,51 @@ def _start_reader_threads(self):
       t.start()
       self.reader_threads.append(t)
 
+  def _get_source_position(self, range_tracker=None, reader=None):
+    if reader:
+      return reader.get_progress().position.byte_offset
+    else:
+      return range_tracker.position_at_fraction(
+          range_tracker.fraction_consumed()) if range_tracker else 0
```

Comment:
<!--new_thread; commit:6e2c396b4cdcce20867c583cba65c8664ff79996; resolved:0-->
Is it valid to return

Reply:
Sorry forgot to complete this.

Should be: Is it valid to return 0 here should that case be an error ?